src/propcache/__init__.py

```diff
@@ -1,6 +1,30 @@
 """propcache: An accelerated property cache for Python classes."""
 
+from typing import TYPE_CHECKING, List
+
 __version__ = "1.0.0"
```

Comment:
Should the major version bump be reverted with this patch?

Reply:
Changed it to 0.2.0.dev0